blog/2023-01-18-free-software-localization.md

```diff
@@ -0,0 +1,72 @@
+
+Tolgee software localization platform is a **free and open-source online tool** that helps software developers localize their products for international markets. Tolgee provides a platform that makes localization easy and affordable with an **easy-to-use interface** for managing translations and supporting all languages. It is constantly being updated with new features and improvements, making it an essential tool for anyone looking to localize their software products. In addition, it provides many great features that make the app localization a piece of cake, such as automatic or machine translation. 
+
+## There are many i18n libraries out there. So why should I use Tolgee?
+
+Tolgee can do a lot more than just traditional i18n library. It doesn't only provide the different strings for different languages, but it also enables you to **change the strings directly in the app you are currently working on**. It also allows your colleagues without coding knowledge to translate/modify/fix strings in your app: even your grandma, aunt, or friend from studies abroad can do that.
+
+Tolgee is, at the same time, a localization platform where you can translate your strings, while it suggests results from various third-party translators like Google Translator, Amazon Translate, or DeepL. It also enables developers to **provide screenshots from the app in one click**. Hence, translators know the context and don't provide wrong translations. 
+
+## How does Tolgee help me to have my App perfectly translated?
+
+The context in which text appears is vital for software localization as it helps translators to understand the meaning of the content and produce an accurate translation. Without context, translations can be misleading or incorrect, leading to user confusion or errors.
+To provide context for translators, Tolgee enables you to provide a **screenshot with a highlighted string** to translate with a single click. In the same dialog, where you can modify your localization strings, you can let Tolgee automatically take the screenshot from your App and highlight the string to be translated. 
+You can also use the [**Tolgee Tools Chrome plugin**](https://chrome.google.com/webstore/detail/tolgee-tools/hacnbapajkkfohnonhbmegojnddagfnj) to translate your App and modify translations live on your page. While translating in the App itself, you can see the location and context of all translations.
+
+## How Tolgee helps developers? (The ALT+click magic)
+There are various dev Tools for various things. Tolgee brings **dev tools for localization**. When you click a translation string in your app with the ALT key, Tolgee dev tools open, and you can translate your string. You never have to edit a .json/.po or other localization data files. The only thing you have to do is ALT + Click the string. Developers won’t be overloaded with localization tasks as even team members without coding skills can manage the localization project independently. 
```

Comment:
Please use newlines in the markdown. Having the paragraphs on single line doesn't make it easy to comment.

Reply:
"Developers won’t be overloaded with" - I wouldn't use this specific word in the development context, since it means overloading of operators for example, which confuses me.